### PR TITLE
docs: Add AWS server

### PR DESCRIPTION
**Pull request by jfoliveira**
Add instructions on how to Add Server to AWS

### CODE REVIEW

File: docs/servers/add-aws-server.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+1. Review server information and settings. If everything is correct, click `CONFIRM`
+    > Use the `PREVIOUS` button if you need to modify any server details
+1. Wait for your server provision to finish
+1. After the server is successfully provisioned it will be visible on `AWS` console
+1. If you want to access the server on `AWS` console:
+    - Access [AWS console](https://console.aws.amazon.com/)
+    - On the Search box, enter “EC2” and select it from search results
+    - In the upper-right corner select the region on which server was provisioned
+    - Click `Instances`
+    - n `EC2` instance with the name provided on Devopness web interface will be listed
```
Comment (Silva97) [inline docs/servers/add-aws-server.md:37]:
```suggestion
    - A `EC2` instance with the name provided on Devopness web interface will be listed
```

File: docs/servers/add-aws-server.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+# Add a Server to Amazon Web Services (AWS)
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `Amazon Web Services` as cloud provider
+1. Select an AWS credential using the drop-down menu
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide
```
Comment (Silva97) [inline docs/servers/add-aws-server.md:21]: This link `../cloud-provider-credentials/get-cloud-providers-credentails.md` is correct? I think it should link the page and not the Markdown file.

File: docs/servers/add-gcp-server.md [hunk 3]
```diff
@@ -13,22 +13,23 @@ links:
...
 1. On Devopness, navigate to a project then select an environment
-1. If there's no other server in the selected environment, just click Add Server
-1. If a list of resource cards is displayed, find the Servers card and click View on it
+1. If there's no other server in the selected environment, just click `ADD SERVER`
+1. If a list of resource cards is displayed, find the `Servers` card and click View on it
 1. On the upper-right corner of the list click `ADD SERVER`
 1. Choose `Google Cloud Platform` as cloud provider
 1. Select a GCP credential using the drop-down menu
-    > If you dont have any GCP credentials listed, follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide before next step
+    > If no credential is listed or you want to use a different one, please follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide
```
Comment (Silva97) [inline docs/servers/add-gcp-server.md:21]: The same case of the link here. It's correct?

**Review by plantaz — APPROVED** [review]

File: docs/servers/add-aws-server.md (hunk 2 above)
Comment (jfoliveira) [reply docs/servers/add-aws-server.md:21]: Nice catch! 🔥

File: docs/servers/add-gcp-server.md (hunk 3 above)
Comment (jfoliveira) [reply docs/servers/add-gcp-server.md]: fixed

**Pull request merged**